layouts/partials/head_custom.html

```diff
@@ -21,3 +21,4 @@
       background-color: black;
   }
 </style>
+<a rel="me" href="https://piaille.fr/@srefrance"></a>
```

Comment:
Il y a un erreur avec l'usage de la balise `<a>` à l'intérieur de la section `<head></head>`.
Je crois, comme mentionné dans la doc de mastodon https://docs.joinmastodon.org/user/profile/#verification que dans ce cas il faudrait utiliser :
```suggestion
<link rel="me" href="https://piaille.fr/@srefrance">
```